README.md

```diff
@@ -4,21 +4,35 @@ Example of embedding Bare in an Android application using <https://github.com/ho
 
 ## Building
 
-To keep the build process fast and efficient, the project relies on a Bare Kit prebuild being available in the [`app/libs/`](app/libs) directory. Prior to building the project, you must therefore either clone and compile Bare Kit from source, or download the latest prebuild from GitHub. The latter is easily accomplished using the [GitHub CLI](https://cli.github.com):
+To keep the build process fast and efficient, the project relies on a Bare Kit prebuild being
+available in the [`app/libs/`](app/libs) directory. Prior to building the project, you must
+therefore either clone and compile Bare Kit from source, or download the latest prebuild from GitHub.
+The latter is easily accomplished using the [GitHub CLI](https://cli.github.com):
 
 ```console
 gh release download --repo holepunchto/bare-kit <version>
 ```
 
-Unpack the resulting `prebuilds.zip` archive and move `android/bare-kit` into [`app/libs/`](app/libs). When finished, either open the project in Android Studio or build it from the commandline:
+Unpack the resulting `prebuilds.zip` archive and move `android/bare-kit` into [`app/libs/`](app/libs).
+When finished, either open the project in Android Studio or build it from the commandline:
 
 ```console
 gradle build
 ```
 
 ### Addons
 
-Native addons will be linked into [`app/src/main/addons/`](app/src/main/addons) as part of the build process and will be automatically included in the final APK bundle by Gradle.
+Native addons will be linked into [`app/src/main/addons/`](app/src/main/addons) as part of the build process
+and will be automatically included in the final APK bundle by Gradle.
+
+## Running
+
+The APK will be located at `./app/build/outputs/apk/<release|debug>`.
```

Comment:
```suggestion
```

That's a Gradle detail I'd rather not leak into the README. The "Run" button in Android Studio is usually a safer bet 😄

Reply:
Yeah I agree on the fact that we should probably encourage people to use the "Run" button, but in the doc we also propose to use `gradle` CLI.